bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Higgs/tHq_4f_ckm_NLO_FXFX_MH125_CP/tHq_4f_ckm_NLO_FXFX_MH125_CP_run_card.dat

```diff
@@ -0,0 +1,144 @@
+# Include the NLO Monte Carlo subtr. terms for the following parton    *
+# shower (HERWIG6 | HERWIGPP | PYTHIA6Q | PYTHIA6PT | PYTHIA8)         *
+# WARNING: PYTHIA6PT works only for processes without FSR!!!!          *
+#***********************************************************************
+  PYTHIA8   = parton_shower
+#***********************************************************************
+# Renormalization and factorization scales                             *
+# (Default functional form for the non-fixed scales is the sum of      *
+# the transverse masses of all final state particles and partons. This *
+# can be changed in SubProcesses/set_scales.f)                         *
+#***********************************************************************
+ F        = fixed_ren_scale  ! if .true. use fixed ren scale
+ F        = fixed_fac_scale  ! if .true. use fixed fac scale
+ 91.188   = muR_ref_fixed    ! fixed ren reference scale 
+ 91.188   = muF1_ref_fixed   ! fixed fact reference scale for pdf1
+ 91.188   = muF2_ref_fixed   ! fixed fact reference scale for pdf2
+#***********************************************************************
+# Renormalization and factorization scales (advanced and NLO options)  *
+#***********************************************************************
+ F        = fixed_QES_scale  ! if .true. use fixed Ellis-Sexton scale
+ 91.188   = QES_ref_fixed    ! fixed Ellis-Sexton reference scale
+ 1        = muR_over_ref     ! ratio of current muR over reference muR
+ 1        = muF1_over_ref    ! ratio of current muF1 over reference muF1
+ 1        = muF2_over_ref    ! ratio of current muF2 over reference muF2
+ 1        = QES_over_ref     ! ratio of current QES over reference QES
+#*********************************************************************** 
+# Reweight flags to get scale dependence and PDF uncertainty           *
+# For scale dependence: factor rw_scale_up/down around central scale   *
+# For PDF uncertainty: use LHAPDF with supported set                   *
+#***********************************************************************
+ .true.   = reweight_scale   ! reweight to get scale dependence
+  0.5     = rw_Rscale_down   ! lower bound for ren scale variations
+  2.0     = rw_Rscale_up     ! upper bound for ren scale variations
+  0.5     = rw_Fscale_down   ! lower bound for fact scale variations
+  2.0     = rw_Fscale_up     ! upper bound for fact scale variations
+  $DEFAULT_PDF_MEMBERS = reweight_PDF     ! reweight to get PDF uncertainty
+#***********************************************************************
+# Merging - WARNING! Applies merging only at the hard-event level.     *
+# After showering an MLM-type merging should be applied as well.       *
+# See http://amcatnlo.cern.ch/FxFx_merging.htm for more details.       *
+#***********************************************************************
+ 3        = ickkw            ! 0 no merging, 3 FxFx merging
```

Comment:
should be "0=ickkw" here, it's a nlo process not matching

Reply:
Hi @menglu21, 

can you please elaborate, why FxFx should not be used here?
 
I wanted to use the settings analogously as in the other SM NLO Higgs cards from the repo, where FxFx merging is used, e.g. [ttH](https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Higgs/tth01j_5f_ckm_NLO_FXFX_MH/tth01j_5f_ckm_NLO_FXFX_MH_run_card.dat#L101) or [ggH](https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Higgs/ggh012j_5f_NLO_FXFX_/ggh012j_5f_NLO_FXFX__run_card.dat#L88).

Reply:
Hi @FloMau if you check the ttH proc card https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Higgs/tth01j_5f_ckm_NLO_FXFX_MH/tth01j_5f_ckm_NLO_FXFX_MH_proc_card.dat#L3, you will find there are two lines in which the first one without additional qcd quark and the second one with additional qcd quark, only in such case we need match/merging. But in you proc card, you don't have such topology so no matching/merging needed

Reply:
Thanks for the explanations, fixed.